### DISCUSSION TITLE
How to view or modify the parameters of the CKD template?

### DISCUSSION THREAD

**Discussion by AlanL526** [Open-Systems-Pharmacology/Forum#1177] [Questions & Problems] [answered]
Dear all,
How do I view or modify the parameters of the CKD template? 
e.g. when I tried to import the simulation using CKD template into MOBI, I found that the volume of kidney in the simulation did not change, but actually the volume of kidney in the CKD template decreased.
Yours,
Sincerely

![1](https://user-images.githubusercontent.com/97157587/158949638-aab7cfd2-cbfb-40d6-89bf-6a5c1cfd9edc.jpg)
![2](https://user-images.githubusercontent.com/97157587/158949659-77311407-c4b6-421f-ba46-910ca26dacf7.jpg)

**Comment by msevestre** [comment]
Hi @AlanL526 

The template is a population template. It does not change the simulation that you use but will be apply on top of a simulation. 

To see the values, you can export the population to CSV, or do as you did already by navigating in the tree and looking at the distributions.

I do not quite understand your second point. The functional kidney volumes are supposed to decrease. Please refer to the paper for more details

**Comment by AlanL526** [comment]
@msevestre 

Sorry for the late reply. What I mean is that, could MOBI view the value of the CKD template?

When I import the simulation run by CKD templates into MOBI, MOBI cannot display the parameter values of CKD template correctly. "Kidney volume" is just an example , and "GFR" is the same. "GFR" in CKD - Stage3 template is defined as 31-60 mL/min/1.73m2, while MOBI still shows the values of healthy individuals.

"To see the values, you can export the population to CSV" works, but I want to add a new compartment to the CKD template via MOBI, which doesn't make much sense if MOBI doesn't show the correct values.

**Reply by msevestre** [reply]
>What I mean is that, could MOBI view the value of the CKD template?

No. A population template/file simply overwrites the parameters of the simulation. 
Say your eGFR is 60 in the simulation. It will be overwritten by the values from the template. 

>but I want to add a new compartment to the CKD template via MOBI

You do not add a compartment to the CKD template. The CKD Template is just a set of individual values.
So to add a compartment, you would add it like you typically do in MoBi. You would parameterize it using the values that make sense for your use case. By running this simulation with the CKD population, you would be able to have your new compartment incorporated in a simulation using the CKD population.

Note: in the upcoming v11, you will be able to create directly CKD individuals and populations.
![image](https://user-images.githubusercontent.com/1041237/159272270-8cc64c63-bc68-4693-a3fd-766aae848dff.png)

**Reply by AlanL526** [reply]
Thank you very much, I appreciate your help!

**Comment by prvmalik** [comment, marked as answer]
Mobi does not run population simulations. I do not know how you would get the CKD simulation into Mobi.

You could build a simulation in PKSim with an individual, export to Mobi to build the structure you wish, and then import back to PKSim for a population simulation with the CKD template?

**Reply by AlanL526** [reply]
Thanks! This method is feasible.
